test/integration/component/Upload_Volume_test_case.py

```diff
@@ -0,0 +1,223 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
@PranaliM conforming to other files, can you rename this file in lowercase, for example `test/integration/component/upload_volume_test_case.py`. Let me know once you do that, I can run a final test run and then we can merge this PR, thanks.

Reply:
@rhtyd , Please check. I have renamed the file to test_ss_volume_usage.py to make it a little more relevant to the scenario being tested.